package.json

```diff
@@ -130,6 +130,8 @@
     "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.33.0",
     "@wordpress/browserslist-config": "^4.0.0",
+    "@wordpress/dependency-extraction-webpack-plugin": "^3.2.1",
+    "@wordpress/scripts": "^19.0.0",
```

Comment:
Could you please why did you need to add this dependency?

Reply:
We're importing the WordPress webpack config from this package. We use this in blocks and other integrations, so I borrowed it for this.

Reply:
Thanks for explaining!